www/index.html

```diff
@@ -223,8 +224,9 @@
   </div>
 </div>
 
-<p>
-  Need help?<br>Ask on <a href="https://discord.gg/rust-lang">#beginners in the Rust Discord</a><br>
+<p id="help">
+  Need help?<br>
+  Ask on <a href="https://discord.gg/rust-lang">#beginners in the Rust Discord</a><br>
```

Comment:
Breaking the second sentence into a separate line is a trivial formatting change, and I'd like to isolate all of those in a separate commit.

Reply:
Reverted.